common/configuration.py

```diff
@@ -0,0 +1,42 @@
+from enum import Enum
+from decouple import config
+
+
+class Environment(Enum):
+    PROD = 'PROD'
+    DEV = 'DEV'
+    TEST = 'TEST'
+
+    @property
+    def is_prod(self) -> bool:
+        return self is self.PROD
+
+    @property
+    def is_dev(self) -> bool:
+        return self is self.DEV
+
+    @property
+    def is_test(self) -> bool:
+        return self is self.TEST
+
+    @classmethod
+    def default(cls) -> 'Environment':
+        return cls.DEV
+
+
+ENVIRONMENT = Environment(config('ENVIRONMENT', default=Environment.default().value).upper())
+
+API_PORT = config('API_PORT', default=None)
+
+LAMBDA_INVOKE_URL = config('LAMBDA_INVOKE_URL', default=None)
+
+DATA_AGGREGATOR_LAMBDA_NAME = config('DATA_AGGREGATOR_LAMBDA_NAME', default=None)
+
+HATHOR_CORE_DOMAIN = config('HATHOR_CORE_DOMAIN', default=None)
+
+HATHOR_NODES = [hn for hn in config('HATHOR_NODES', default='').split(',') if hn != '']
```

Comment:
Have you tried this?
```
from decouple import Csv
HATHOR_NODES = config('HATHOR_NODES', cast=Csv())
```

It is decouple`s built-in way of doing the same thing.
But I'm also ok with the current solution.

Reply:
Awesome!!!

Reply:
done!